packages/vue-searchbox/examples/demo/package.json

```diff
@@ -10,6 +10,7 @@
     "dependencies": {
         "@appbaseio/vue-searchbox": "1.2.0",
         "core-js": "^2.6.5",
+        "hotkeys-js": "^3.8.7",
```

Comment:
@mohdashraf010897 Remove it. We should not modify the existing example.

Reply:
ok 👍